### PR TITLE
updated the source code url to reflect repo name change

### PR DESCRIPTION
**Pull request by iammoen**
Signed-off-by: Chris Moen <iammoen@gmail.com>

## Description
The repository at gitlab that the Source Code url pointed to has moved under a new name thus changing the url. While the old one functions and redirects I believe it appropriate to update to the new location.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: distributions/distribution-list.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Along with the [offical Chef distributions](https://downloads.chef.io/) of our O
 
 This includes community distributions of Chef's software (including Chef Infra, InSpec, Habitat, and Automate).
 
-* [Source Code](https://gitlab.com/cc-build)
+* [Source Code](https://gitlab.com/cinc-project)
 * [Builds](http://downloads.cc-build.org/)
```
Comment (ramereth) [inline distributions/distribution-list.md]: Can you please also update this URL?
```suggestion
 * [Builds](http://downloads.cinc.sh/)
```

File: distributions/distribution-list.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ Along with the [offical Chef distributions](https://downloads.chef.io/) of our O
 
 This includes community distributions of Chef's software (including Chef Infra, InSpec, Habitat, and Automate).
 
-* [Source Code](https://gitlab.com/cc-build)
-* [Builds](http://downloads.cc-build.org/)
+* [Source Code](https://gitlab.com/cinc-project)
+ * [Builds](http://downloads.cinc.sh/)
```
Comment (ramereth) [inline distributions/distribution-list.md]: Oops, looks like my suggestion added an extra leading whitespace.
```suggestion
* [Builds](http://downloads.cinc.sh/)
```

File: distributions/distribution-list.md (hunk 1 above)
Comment (iammoen) [reply distributions/distribution-list.md]: Thats a great catch.

**Comment by tas50** [comment]
Thanks @iammoen

**Pull request merged**